.github/workflows/build-and-publish.yaml

```diff
@@ -0,0 +1,44 @@
+---
+name: Build and Publish
+
+on:
+  push:
+    branches:
+      - redhat-** # IMPORTANT! this must match the jobs.build-and-publish.env.BRANCH_PREFIX (save the **).
+
+jobs:
+  build-and-publish:
+    name: Publish container image
+    env:
+      BRANCH_PREFIX: redhat- # IMPORTANT! this must match the .on.push.branches prefix!
+      REGISTRY: quay.io/projectquay
+      REPO_NAME: ${{ github.event.repository.name }}
```

Comment:
Is this `quay`, or `quay/quay`. In either case, is `${{ env.REGISTRY }}/${{ env.REPO_NAME }}:${{ env.TAG }}` below correct?

Otherwise LGTM